src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Configuration/TargetFrameworkProjectConfigurationDimensionProviderTests.cs

```diff
@@ -71,7 +75,8 @@ public class TargetFrameworkProjectConfigurationDimensionProviderTests
             using (var projectFile = new MsBuildProjectFile(projectXmlTFMs))
             {
                 IProjectXmlAccessor _projectXmlAccessor = IProjectXmlAccessorFactory.Create(projectFile.Project);
-                var provider = new TargetFrameworkProjectConfigurationDimensionProvider(_projectXmlAccessor);
+                ITelemetryService _telemetryService = ITelemetryServiceFactory.Create();
```

Comment:
I replied below and I completely agree. There's an issue to create a VSI test for config changes and I testing the telemetry for the config changes will go there.